test/jdk/tools/jpackage/helpers/jdk/jpackage/test/Executor.java

```diff
@@ -324,8 +339,17 @@ private Path executablePath() {
         return executable.toAbsolutePath();
     }
 
+    private List<String> prefixCommandLineArgs() {
+        if (winEnglishOutput) {
+            return List.of("cmd.exe", "/c", "chcp", "437", ">nul", "2>&1", "&&");
```

Comment:
I assume `chcp 437` works as expected. The summary still references `chcp 65001` command.

Reply:
Yes, I confirmed that it works as expected. 

> The summary still references chcp 65001 command.

Thanks, I fixed summary.